app/pkg/email/smtp/smtp.go

```diff
@@ -157,3 +168,15 @@ func sendMail(localName, serverAddress string, a gosmtp.Auth, from string, to []
 	}
 	return c.Quit()
 }
+
+func generateMessageID() (string, error) {
+	timestamp := strconv.FormatInt(time.Now().UTC().UnixNano(), 10)
+	unq := uuid.New()
```

Comment:
I'd prefer to use `rand.String(32)` that is already available on Fider. This random string + timestamp is unique enough for this operation. By doing so, we won't need the external `uuid` package.